From 0870ab88262bb61124899ad98d06a72a6cdc7129 Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Fri, 19 Jun 2015 11:56:56 +0200
Subject: [PATCH] sdpops: don't return 0 if no match for
 sdp_get_line_startswith()

- that causes exit of config execution
- reported by Sebastian Damm
---
 modules/sdpops/sdpops_mod.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/sdpops/sdpops_mod.c b/modules/sdpops/sdpops_mod.c
index 2a4aa6b..f853996 100644
--- a/modules/sdpops/sdpops_mod.c
+++ b/modules/sdpops/sdpops_mod.c
@@ -1657,7 +1657,7 @@ static int w_sdp_get_line_startswith(sip_msg_t *msg, char *avp, char *s_line)
 		p = find_sdp_line(line.s + line.len, body.s + body.len, sline.s[0]);
 	}
 
-	return 0;
+	return -1;
 }
 
 /**
-- 
1.9.1

